language-design.md

```diff
@@ -16,6 +16,44 @@ are:
 For each quote, describe what it was about the quote that led you pick it.
 
 **Response**
+Anthony
+1. Cathedral/Bazaar Quote
+
+"The key point is that in the bazaar style of building a program or designing a language or what you will,
```

Comment:
For this section, use Markdown blockquotes for the quotes so they are formatted like so:

> "This is a block quote"

Which can be achieved with the following Markdown:
```
> "This is a block quote"
```

(Preceed the lines with a `>` character.)